components/Embed/index.tsx

```diff
@@ -35,7 +35,18 @@ const Embed = ({
   ...attrs
 }: EmbedProps) => {
   if (typeof iframe !== 'boolean') iframe = iframe === 'true';
-  if (html === 'false') html = undefined;
+
+  if (html) {
+    try {
+      if (html !== decodeURIComponent(html)) {
+        html = decodeURIComponent(html);
+      } else if (html === 'false') {
+        html = undefined;
+      }
+    } catch (e) {
+      if (html === 'false') html = undefined;
```

Comment:
i know this isn't really doing anything and a catch block can be empty, but it still feels weird 😂